sdk/search/search/review/search.api.md

```diff
@@ -868,6 +912,17 @@ export interface SimpleField {
     type: SimpleDataType;
 }
 
+// @public
+export interface Skillset {
+    // Warning: (ae-forgotten-export) The symbol "CognitiveServicesAccount" needs to be exported by the entry point index.d.ts
+    cognitiveServicesAccount?: CognitiveServicesAccount;
+    description: string;
+    etag?: string;
+    name: string;
+    // Warning: (ae-forgotten-export) The symbol "Skill" needs to be exported by the entry point index.d.ts
```

Comment:
here too

Reply:
Checked and confirmed no more of them is present.